src/protocols/bitcoin/helpers/index.ts

```diff
@@ -3,3 +3,19 @@ import { BTC_COIN_PRECISION } from '../config';
 export function satoshiToBtc(amount: number) {
   return amount / 10 ** BTC_COIN_PRECISION;
 }
+
+export function normalizeTransactionStructure(t: any, transactionOwner?: string) {
```

Comment:
I strongly advise against using single letter variables. It's really hard to debug/refactor/follow them.